common/templates/discussion_mathjax_include.html

```diff
@@ -0,0 +1,45 @@
+## mako
+## File:   templates/mathjax_include.html
```

Comment:
@singingwolfboy I added this just for some testing purpose I will update the line